canonical-web-and-design/canonical.com
# Add search input with JS fuzzy match

## Done
- Install `fuse.js`
- Enable search

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- 

## Issue / Card

Fixes #
